File: program/program/convert-hours-to-seconds/ConvertHoursToSeconds.ts
function HoursToSeconds(hours: number) {

    // 1 hour = 60 minutes;
    // 1 minute = 60 seconds;
    // 1 hour = 60 mins * 60sec = 3600 seconds
 
    const minutes = hours * 60;
    const seconds = minutes * 60;
    return seconds;

}

console.log(HoursToSeconds(5) + " seconds");

File: program/program/convert-hours-to-seconds/convert_hours_to_seconds.go
// Created by: Ravi Teja
// Language: GO
// Github: https://github.com/Sairaviteja27

// Declare Main Package
package main

// Import input output package, lets us use things such as Print
import "fmt"

// Main Function
func main() {
	var hours int
	fmt.Println("Enter number of hours :")
	fmt.Scan(&hours)
	fmt.Println("Number of seconds is ", hours*3600)
}
